renderer-macos/lib/components/TextInput.re

```diff
@@ -0,0 +1,37 @@
+open Brisk;
+
+type attribute = [ Layout.style ];
+
+type style = list(attribute);
+
+let component = nativeComponent("TextInput");
+let make = (~style= [], ~callback=() => (), children) =>
+  component((_: Hooks.empty) =>
+    {
+     make: () => {
+       let input =
+         BriskTextInput.(
+           make(~onChange=callback, ())
+         );
+     },
+     configureInstance: (~isFirstRender as _, {view} as node) => {
+       style
+       |> List.iter(attribute =>
```

Comment:
just FYI - this doesn't do any magic. In this case the iter does nothing.

Reply:
Good to know, thanks. 😅
I have so many errors being thrown regarding `BriskTextInput.c`, I feel like once I get that ironed out I'll have a better way of understanding and implementing the component part.